api/routes/story.route.js

```diff
@@ -26,22 +26,17 @@ MongoClient.connect('mongodb://localhost:27017/',
 });
 
 storyRoutes.route('/getStoryById/:id').get((req, res) => {
-    Story.findOne({id: req.params.id}, (err, story) => {
-        if(err) {
-          console.log(err);
-          res.json(err);
-        }
-        if(story) res.json(story);
-    });
-});
-
-storyRoutes.route('/getStoryByUnderscoreId/:id').get((req, res) => {
-    Story.findOne({_id: req.params.id}, (err, story) => {
+    Story.findById(req.params.id, (err, story) => {
         if(err) {
-          console.log(err);
-          res.json(err);
+            console.log(err);
+            res.status(400).send("An error occurred while trying to find this profile");
+        } else {
+            if(story) {
+                res.status(200).json(story);
+            } else {
+                res.status(404).send("Story with given ID not found");
+            }
```

Comment:
We can get rid of these `else`s as `res.status` effectively `return`s!

```suggestion
if(!story) {
    res.status(404).send("Story with given ID not found");
}
res.status(200).json(story);             
```